app/cafe.py

```diff
@@ -0,0 +1,18 @@
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not "vaccine" in visitor:
+            raise NotVaccinatedError

```

Comment:
The `NotVaccinatedError` should be raised with a descriptive message. According to the checklist, you should provide a message when raising exceptions. For example: `raise NotVaccinatedError('Visitor is not vaccinated.')`.